index.html

```diff
@@ -14,58 +14,69 @@
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="styles.css">
+
 </head>
 <body>
-  <div class="Container">
-    <header class="Mylogo">
-      <div class="left-logo"> 
-        <a class="logotitle" href="#homepage">
-        <span class="logo">My Logo</span>
-        </a>
-      </div>
-      <nav class="right-logo">
-        <a class="hidden" href="#homepage">Portfolio</a>
-        <a class="hidden" href="#homepage">About</a>
-        <a class="hidden" href="#homepage">Contact</a>
-        <button class="Hambu-button">
-        <a><img src="Union.png"></a>
-        </button>
-      </nav>
-    </header>
-  <div class="img">
-    <section class="bio">
-      <header class="bio1">
-        <h1 id="title">I’m Mariane Glad to see you!</h1>
-        <h1 id="title3">I’m Alexandra<br>Glad to see you!</h1>
-        <p id="paragraph">I’m a software developer! I can help you build a product , 
-      feature or website Look through some of my work and experience! 
-      If you like what you see and have a project you need coded, 
-      don’t hestiate to contact me.</p>
-        <h3 id="connect">LET’S CONNECT</h3>
-    </header>
-    </section>
-    <nav class="bottom-icons">
-      <ul>
-        <li>
-          <a href="#"><img src="twitter.png" alt="twitter"></a>
-        </li>
-        <li>
-          <a href="#"><img src="Linkedin icon.png" alt="linkedin"></a>
-        </li>
-        <li>
-          <a href="#"><img src="m.png" alt="medium"></a>
-        </li>
-        <li>
-          <a href="#"><img src="github.png" alt="github"></a>
-        </li>
-        <li>
-          <a href="#"><img src="hand.png" alt="union"></a>
-        </li>
-      </ul>
-    </nav>
-  </div>
+  <div id="Purple"> 
+    <div class="x-btn"> 
+      <button type="btn-close" id="close" >
+        <img src="/images/Normal Button/Tertiary/Icons/Normal Button/Tertiary/Icons/close.png" alt="x">
+      </button>
+    </div>
+    <a class="pop-window" href="#Portfolio">Portfolio</a>
+    <a class="pop-window" href="#about">about</a>
+    <a class="pop-window" href="#footer"> conctact</a>   
   </div>
```

Comment:
You did well Implementing this but it would be nice if you Paid more attention to detail always type correct spellings for this is a representation of yourself you dont want to confuse people that will view your Portfolio by typing wrong spellings and Also you should always start a word with a capital letter so your "about" should be written as "About" and conctact should be written as "Contact". Please stick to the template design 

This is your design:

 
![david mobile ](https://user-images.githubusercontent.com/79506105/194969578-917357b8-9c81-4c95-894a-32e5c240721e.png)

Please Your design should look like this:
![mobile 1](https://user-images.githubusercontent.com/79506105/194969629-49ca8c9a-9ae5-4f7c-890d-bf4a4216b0a2.png)

2. Another issue I notice is the gap between Portfolio, About and Contact is small please adjust it so it looks similar to the design on figma you can do this by adjusting your margin-top to 10%. Adjust your font size too.

Please lets Pay attention to detail to avoid making mistakes.